node-reinstall

```diff
@@ -182,12 +182,18 @@ sudo rm -rf $PREFIX/bin/{node,npm}
 sudo rm -rf $PREFIX/share/man/man1/node.1
 sudo rm -rf $PREFIX/lib/dtrace/node.d
 
+# remove global node_modules on arch
+[ -d /usr/lib/node_modules ] && sudo rm -rf /usr/lib/node_modules
+
 if (( $USE_NVM )); then
   # go home and install NVM just because I feel safe there
   cd $HOME
-  # get the latest stable version number of nvm from the repo's homepage
-  [ "$STABLE" == "" ] && STABLE=$(curl -s -k https://github.com/creationix/nvm/ | grep "curl https://raw.githubusercontent.com/creationix/nvm/" | grep -oE "v\d+\.\d+\.\d+")
-  [[ $STABLE =~ ^v[0-9]+.[0-9]+.[0-9]+$ ]] || STABLE="v0.25.1"
+  # get the latest stable version number of nvm from the repo's package.json
```

Comment:
This is very brittle. The previous approach is the one you should be using (getting a list of tags, sorting, and grabbing the last one).

Reply:
Thanks for taking the time to review this, @ljharb.

The previous version was just checking the README for the version, whereas this version gets the package.json and parses the JSON, which is arguably a bit safer than parsing a Markdown.

Having said that, I prefer the idea of fetching the tags on your repo and getting the latest stable version.

I'll look into this as an option. We are using your `curl` instructions now to just pull down the installer, but I think we could clone the repo and checkout the latest stable tag.

Thanks for the feedback.